### PR TITLE
Nothing to do on these WEB_X modules

### PR DESCRIPTION
**Pull request by gfcapalbo**
Description of the issue/feature this PR addresses:
Tagging web_calendar , web_diagram  , web_kanban   ,web_kanban_gauge  as  'nothing to do'

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/openupgrade/doc/source/modules90-100.rst [hunk 1]
```diff
@@ -457,15 +457,15 @@ Status :
 +-----------------------------------+-----------------------------------+
 |web_analytics                      |                                   |
 +-----------------------------------+-----------------------------------+
-|web_calendar                       |                                   |
+|web_calendar                       | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|web_diagram                        |                                   |
+|web_diagram                        | Nothing to do                     |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules90-100.rst:488]: As this module contains a migrations folder, it's convenient to include the openupgrade_analysis_work.txt file indicating that there's nothing to do. This way, looking at the folder structure without checking this documentation, you can see if the module has been analyzed.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I have checked folders and run the script for looking for noupdate changes. One a little comment that I think it's important.

**Review by pedrobaeza — APPROVED** [review]
Well, this is normally performed copying the analysis file, and adding on the corresponding sections "Nothing to do" for reviewers to check if it's that way, but it's very simple, so I approve it.

Travis error is unrelated and should be fixed when migrating `crm`.

**Comment by gfcapalbo** [comment]
Fixed the work comments according to usual standards and rebased.

**Comment by pedrobaeza** [comment]
There's something strange in the rebase. Please check.

**Comment by gfcapalbo** [comment]
conflict solved

**Comment by pedrobaeza** [comment]
I still see a lot of commits that don't belong here

**Comment by hbrunn** [comment]
this branch still needs cleaning up

File: odoo/openupgrade/doc/source/modules90-100.rst (hunk 1 above)
Comment (gfcapalbo) [reply odoo/openupgrade/doc/source/modules90-100.rst:488]: done

**Comment by gfcapalbo** [comment]
cannot find anything wrong except for the travis CRM, cleaned up from a fresh OCA 10.0 and cherry picked my small commit.

**Comment by hbrunn** [comment]
you didn't use a pristine 10.0 branch, but some other one that contains the extra commits @pedrobaeza and me are talking about

**Pull request merged**